mbtools/html_to_json.py

```diff
@@ -1,22 +1,48 @@
 import argparse
-import glob
+import os
 import json
 from pathlib import Path
 
 
-def create_json_content(uuid, content, variant="main"):
+def create_json_content(uuid, content):
+
     json_content = {"id": uuid, "content":
-                    [{"variant": variant, "html": content}]}
+                                content}
     return json.dumps(json_content, indent=2)
 
 
 def html_to_json(html_directory, json_directory):
-    for file_name in glob.iglob(f'{html_directory}/*.html'):
-        with open(f'{file_name}') as f:
-            new_file = open(f'{json_directory}/'
-                            f'{Path(file_name).stem}.json', "w")
-            new_file.write(create_json_content(Path(file_name).stem, f.read()))
-            new_file.close()
+
+    for file_name in os.listdir(html_directory):
+        if os.path.isfile(f'{html_directory}/{file_name}'):
+
+            file_uuid = f'{Path(file_name).stem}'
+            file_content = ''
+
+            with open(f'{html_directory}/{file_name}') as f:
+                file_content = f.read()
+
+            with open(f'{json_directory}/'
+                      f'{Path(file_name).stem}.json', "w") as new_file:
+                new_file.write(create_json_content(
+                    file_uuid, [{"variant": "main", "html": file_content}]))
+
+        else:
```

Comment:
I think we're needlessly adding this branching. Per my comment on the test, we should only consider variant files where a "main" variant exists. So in this loop if `path.is_file()` returns `False` you can just continue. Then for each "main" file you can explicitly check for a directory `html_directory/file_uuid` and populate variants in a single pass. I think structuring things that way would also simplify the nested conditionals a bit since you could create a `content` array, append with any variants found, and then make the call to `create_json_content`.